### ISSUE TITLE
Nullpointer on X-Parse-App-Display-Version when running instrumentation tests

### ISSUE THREAD

**Issue by abraaocaldas** [parse-community/Parse-SDK-Android#1012]
Hello all,

When I try to use Parse Android SDK on instrumentation tests it always fail on the first call for example on login :

A simple

```
ParseUser.logIn( "user",
                "password"
            )
```

Will return this error:

```
java.lang.NullPointerException: value for name X-Parse-App-Display-Version == null
at okhttp3.Headers.checkValue(Headers.java:268)
at okhttp3.Headers$Builder.set(Headers.java:384)
at com.parse.ParsePlugins$1.intercept(ParsePlugins.java:127)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:257)
at okhttp3.RealCall.execute(RealCall.java:93)
at com.parse.ParseHttpClient.executeInternal(ParseHttpClient.java:71)
at com.parse.ParseHttpClient.execute(ParseHttpClient.java:57)
at com.parse.ParseRequest$3.then(ParseRequest.java:133)
at com.parse.ParseRequest$3.then(ParseRequest.java:130)
at bolts.Task$15.run(Task.java:917)
at bolts.BoltsExecutors$ImmediateExecutor.execute(BoltsExecutors.java:105)
at bolts.Task.completeAfterTask(Task.java:908)
at bolts.Task.continueWithTask(Task.java:715)
at bolts.Task.continueWithTask(Task.java:726)
at bolts.Task$13.then(Task.java:818)
at bolts.Task$13.then(Task.java:806)
at bolts.Task$15.run(Task.java:917)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
at java.lang.Thread.run(Thread.java:919)

```

I'm doing the initialization as documentation says. If I try the same code on live code (normal APK) it works fine, but on instrumentation test it don't work at all.

```
/*Parse SDK*/
    implementation "com.github.parse-community.Parse-SDK-Android:parse:1.23.1"
    implementation "com.github.parse-community.Parse-SDK-Android:ktx:1.23.1"

 defaultConfig {
        minSdkVersion 26
        targetSdkVersion 29
        versionCode 1
        versionName "1.0"

        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
        consumerProguardFiles 'consumer-rules.pro'
    }
```

Any help is appreciated!

Best Regards,

Abraão Caldas

**Comment by Jawnnypoo** [comment]
Will look into fixing this in general, but the line is https://github.com/parse-community/Parse-SDK-Android/blob/0cfc67e9a3bcedc714f0041ee60cafb40ed93520/parse/src/main/java/com/parse/ParsePlugins.java#L127 which you could likely mock out `ManifestInfo.getVersionName` and having it return something non-null for now

**Comment by mtrezza** [comment]
@L3K0V I assume this has been closed via #1014?

**Comment by L3K0V** [comment]
> @L3K0V I assume this has been closed via #1014?

 Correct.
